src/server/game/Entities/GameObject/GameObject.cpp

```diff
@@ -1432,6 +1432,32 @@ void GameObject::UseDoorOrButton(uint32 time_to_restore, bool alternative /* = f
     m_cooldownTime = time_to_restore ? (GameTime::GetGameTimeMS() + time_to_restore) : 0;
 }
 
+void GameObject::PlayAnimKit(int32 animKit)
+{
+    WorldPacket data(SMSG_GAME_OBJECT_ACTIVATE_ANIM_KIT, 4 + 2);
```

Comment:
Opcode does not exist in 3.3.5.

Anim kit is set via `GAMEOBJECT_BYTES_1[1]`

Reply:
No, since the script action enum for 3.3.5 ends before anim kits, this function isnt needed at all

Reply:
Yes i'm retarded and keep getting confused between art and anim